docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  mongodb:
+    image: mongo:latest
+    volumes:
+      - /your/data/location:/data/db
```

Comment:
`/your/data/location` is just a placeholder string here? The [reference](https://docs.docker.com/compose/compose-file/compose-file-v2/#volume-configuration-reference) suggests that it may be best practice to have a top-level named volume (`database`?) and mount it here as `database:/data/db`.